tidb-cloud/serverless-export.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Export Data from TiDB Serverless
+summary: Learn how to export data from TiDB Serverless clusters.
+---
+
+# Export Data from TiDB Serverless
+
+TiDB Serverless Export (Beta) is a service that enables you to export data from a TiDB Serverless cluster to local storage or an external storage service. You can use the exported data for backup, migration, data analysis, or other purposes.
+
+While you can also export data using tools such as [mysqldump](https://dev.mysql.com/doc/refman/8.0/en/mysqldump.html) and TiDB [Dumpling](https://docs.pingcap.com/tidb/dev/dumpling-overview), TiDB Serverless Export offers a more convenient and efficient way to export data from a TiDB Serverless cluster. It brings the following benefits:
+
+- Convenience: the export service provides a simple and easy-to-use way to export data from a TiDB Serverless cluster, eliminating the need for additional tools or resources.
+- Isolation: the export service uses separate computing resources, ensuring isolation from the resources used by your online services.
+- Consistency: the export service ensures the consistency of the exported data without causing locks, which does not affect your online services.
+
+## Features
+
+This section describes the features of TiDB Serverless Export.
+
+### Export location
+
+You can export data to local storage or [Amazon S3](https://aws.amazon.com/s3/).
+
+> **Note:**
+>
+> If the size of the data to be exported is large (more than 100 GiB), it is recommended that you export it to Amazon S3.
+
+**Local storage**
+
+Exporting data to local storage has the following limitations:
+
+- Exporting multiple databases to local storage at the same time is not supported.
+- Exported data is saved in the stashing area and will expire after two days. You need to download the exported data in time.
+- TiDB Cloud offers 250 GiB of storage space in the stashing area for each organization per region. If the storage space is full, you will not be able to export data to local storage.
+
+**Amazon S3**
+
+To export data to Amazon S3, you need to provide an [access key](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html) for your S3 bucket. Make sure the access key has read and write access for your S3 bucket, including at least these permissions: `s3:PutObject` and `s3:ListBucket`.
+
+### Data filtering
+
+You can filter data by specifying the database and table you want to export. If you specify a database without specifying a table, all tables in that specified database will be exported. If you do not specify a database when you export data to Amazon S3, all databases in the cluster will be exported.
+
+> **Note:**
+>
+> You must specify the database when you export data to local storage.
+
+### Data formats
+
+You can export data in the following formats:
```

Comment:
I think we should explain more about the exported formats. https://docs.pingcap.com/tidb/stable/dumpling-overview#format-of-exported-files